cocos/tween/actions/action-interval.ts

```diff
@@ -522,28 +529,33 @@ export class Repeat extends ActionInterval {
     clone (): Repeat {
         const action = new Repeat();
         this._cloneDecoration(action);
-        action.initWithAction(this._innerAction!.clone(), this._times);
+        if (this._innerAction) {
+            action.initWithAction(this._innerAction.clone(), this._times);
+        }
         return action;
     }
 
-    startWithTarget (target: any): void {
+    startWithTarget<T> (target: T | null): void {
         this._total = 0;
-        this._nextDt = this._innerAction!._duration / this._duration;
-        ActionInterval.prototype.startWithTarget.call(this, target);
-        this._innerAction!.startWithTarget(target);
+        this._nextDt = (this._innerAction ? this._innerAction._duration : 0) / this._duration;
+        super.startWithTarget(target);
```

Comment:
Uses typescript `super.` instead of javascript `prototype.`

Reply:
Good catch, it may be 0 since its initialized value is 0. I will add protection logic for 0.

Reply:
Update. I think `this._duration` is not possiable to be `0`. Look at this logic:

```ts
export class Repeat extends ActionInterval {
    private _times = 0;
    private _total = 0;
    private _nextDt = 0;
    private _actionInstant = false;
    private _innerAction: FiniteTimeAction | null = null;

    constructor (action?: FiniteTimeAction, times?: number) {
        super();
        this.initWithAction(action, times); // It invokes initWithAction in the constructor
    }

    /*
     * @param {FiniteTimeAction} action
     * @param {Number} times
     * @return {Boolean}
     */
    initWithAction (action?: FiniteTimeAction, times?: number): boolean {
        if (!action || times === undefined) {
            return false;
        }
        const duration = action._duration * times;

        if (this.initWithDuration(duration)) { // It invokes initWithDuration here even if  duration is 0
            this._times = times;
            this._innerAction = action;
            if (action instanceof ActionInstant) {
                this._actionInstant = true;
                this._times -= 1;
            }
            this._total = 0;
            return true;
        }
        return false;
    }

    initWithDuration (d: number): boolean {
         // Look at this line, if d is zero, this._duration will be assigned to  FLT_EPSILON which is a small number.
        this._duration = (d === 0) ? macro.FLT_EPSILON : d;
        // prevent division by 0
        // This comparison could be in step:, but it might decrease the performance
        // by 3% in heavy based action games.
        this._elapsed = 0;
        this._firstTick = true;
        return true;
    }
```